doc/api/cli.md

```diff
@@ -167,6 +167,18 @@ Enable experimental Source Map V3 support for stack traces.
 Currently, overriding `Error.prepareStackTrace` is ignored when the
 `--enable-source-maps` flag is set.
 
+### `--experimental-abortcontroller`
+<!-- YAML
+added: REPLACEME
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/33527
+    description: --experimental-abortcontroller is no longer required.
+-->
+
+Experimental `AbortController` and `AbortSignal` support is enabled by default.
+Use of this command line flag is no longer required.
```

Comment:
I guess that means that this flag + all the documentation around it can be removed entirely?

Reply:
The flag itself is still there as a non-op. Given that it will be required in 14.x, I think there's value in keeping the documentation around at least while the API is still considered experimental. Shouldn't be long until we can remove it.

Reply:
The unflag commit should remove the flag then and replace it with a `NoOp{}` flag, like e.g. `--experimental-worker`.